### PR TITLE
Update Windows build instructions and .gitignore

### PR DESCRIPTION
**Pull request by JosJuice**
This change is [Reviewable](https://reviewable.io/reviews/dolphin-emu/dolphin/3785)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,6 +18,8 @@ Source/Core/Common/scmrev.h
 *.vcxproj.user
 *.obj
 *.tlog
+*.VC.opendb
+*.VC.db
```
Comment (BhaaLseN) [inline .gitignore]: Maybe add the VS2015 `.vs/` folder while we're at it?

**Comment by JosJuice** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---

*[.gitignore, line 22 \[r1\]](https://reviewable.io:443/reviews/dolphin-emu/dolphin/3785#-KFUcSn4bKkoAsSlseqM-r1-22:-KFUfd4f246YdjwysoXZ:-499146009) ([raw file](https://github.com/dolphin-emu/dolphin/blob/4836a98820a8c8385eb71567f7b785d4d7940cf6/.gitignore#L22)):*
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/dolphin-emu/dolphin/3785)*

**Comment by BhaaLseN** [comment]
:lgtm:

---

Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/dolphin-emu/dolphin/3785#-:-KFUg4dK0LsxhGsn4Rab:1417088721)*

**Comment by mimimi085181** [comment]
You don't need to install the Windows 10 SDK manually. You just need to check the "Universal Windows App Development Tools" option in the Visual Studio installer.

**Comment by JosJuice** [comment]
Yes, but that doesn't change the fact that the Windows 10 SDK needs to be installed. Installing it during the Visual Studio installation process is just a different way of installing it.

**Pull request merged**